vcpkg/readme/vcpkg-tool-README.md

```diff
@@ -0,0 +1,117 @@
+| `artifact source` | Also known as a “feed”. An Artifact Source is a location that hosts metadata to locate artifacts. (_There is only one source currently_) |
+| `activation` | The process by which a particular set of artifacts are acquired and enabled for use in a calling command program.|
+| `versions` | Version numbers are specified using the Semver format. If a version for a particular operation isn't specified, a range for the latest version ( `*` ) is assumed. A version or version range can be specified using the npm semver matching syntax. When a version is stored, it can be stored using the version range specified, a space and then the version found. (ie, the first version is what was asked for, the second is what was installed. No need for a separate lock file.) |
+
+
+# Contributing
+
+Please refer to the "contributing" section of the
+[main `README.md`](https://github.com/microsoft/vcpkg/blob/master/README.md).
+
+This project has adopted the [Microsoft Open Source Code of Conduct][contributing:coc].
+For more information see the [Code of Conduct FAQ][contributing:coc-faq]
+or email [opencode@microsoft.com](mailto:opencode@microsoft.com)
+with any additional questions or comments.
+
+[contributing:submit-issue]: https://github.com/microsoft/vcpkg/issues/new/choose
+[contributing:submit-pr]: https://github.com/microsoft/vcpkg/pulls
+[contributing:coc]: https://opensource.microsoft.com/codeofconduct/
+[contributing:coc-faq]: https://opensource.microsoft.com/codeofconduct/
+
+## Windows Contributing Prerequisites
+
+* Install Visual Studio and the C++ workload
+* Install Node.JS by downloading a 16.x copy from https://nodejs.org/en/
+* `npm install -g @microsoft/rush`
+
+## Ubuntu 22.04 Contributing Prerequisites
+
+```
+curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
+sudo apt update
+sudo apt install nodejs cmake ninja-build gcc build-essential git zip unzip
+sudo npm install -g @microsoft/rush
+```
+
+# License
+
+The product code in this repository is licensed under the [MIT License](LICENSE.txt). The tests
+contain 3rd party code as documented in `NOTICE.txt`.
+
+# Trademarks
+
+This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft
+trademarks or logos is subject to and must follow
+[Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
```

Comment:
```suggestion
[Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
```
Suggestion: remove the en-us from the URL so users of other languages can easily view in their preferred language.